saleor/graphql/views.py

```diff
@@ -26,6 +29,27 @@
 handled_errors_logger = logging.getLogger("saleor.graphql.errors.handled")
 
 
+def tracing_wrapper(execute, sql, params, many, context):
+    if not settings.ENABLE_OPENTRACING:
+        return execute(sql, params, many, context)
+
+    start = time.time()
+    with opentracing.tracer.start_span(operation_name="query") as span:
+        span.set_tag("sql", sql)
```

Comment:
See the [naming conventions](https://github.com/opentracing/specification/blob/master/semantic_conventions.md) for standard tag names.